internal/rpc/user_devices.go

```diff
@@ -448,6 +459,15 @@ func (s *userDeviceRPCServer) deviceModelToAPI(ud *models.UserDevice) *pb.UserDe
 				Expiration: timestamppb.New(vc.ExpirationDate),
 			}
 		}
+
+		if sd := vnft.R.VehicleTokenSyntheticDevice; sd != nil {
```

Comment:
I think we should be careful here. There is a state in between where the synthetic device is getting minted but has not acquired a token id yet. Can we only populate this when the token id is populated? I think it’s going to be confusing to clients.

Is it going to matter a lot? Probably not, but it will happen.